pdns/ws-auth.cc

```diff
@@ -896,19 +901,17 @@ static bool isValidMetadataKind(const string& kind, bool readonly) {
     "SLAVE-RENOTIFY",
     "SOA-EDIT",
     "TSIG-ALLOW-AXFR",
-    "TSIG-ALLOW-DNSUPDATE"
-  };
+    "TSIG-ALLOW-DNSUPDATE"};
 
   // the following options do not allow modifications via API
-  static vector<string> protectedOptions {
+  static vector<string> protectedOptions{
     "API-RECTIFY",
     "AXFR-MASTER-TSIG",
     "NSEC3NARROW",
     "NSEC3PARAM",
     "PRESIGNED",
     "LUA-AXFR-SCRIPT",
-    "TSIG-ALLOW-AXFR"
-  };
+    "TSIG-ALLOW-AXFR"};
```

Comment:
this formatting surprises me - it reduces the quality of diffs

but, that's not a discussion for here on this PR I think

Reply:
I fixed it in #13663